v1/customer/customer.gunk

```diff
@@ -0,0 +1,151 @@
+//                                 Description:      "Access to transactions resources is restricted to authenticated users. The authentication process follows the Authorization Code flow.",
+//                                 AuthorizationURL: "/v1/auth",
+//                                 TokenURL:         "/v1/token",
+//                                 Scopes: openapiv2.Scopes{
+//                                         Scope: map[string]string{
+//                                                 "read":  "Grants read access",
+//                                                 "write": "Grants write access",
+//                                         },
+//                                 },
+//                         },
+//                 },
+//         },
+// }
+package customer
+
+import (
+	"time"
+
+	"github.com/gunk/opt/http"
+	"github.com/gunk/opt/openapiv2"
+
+	types "github.com/openbank/openbank/v1/types"
+)
+
+type FaceImage struct {
+	URL  string    `pb:"1" json:"url"`
+	Date time.Time `pb:"2" json:"date"`
+}
+
+type CreditRating struct {
+	Rating string `pb:"1" json:"rating"`
+	Source string `pb:"2" json:"source"`
+}
+
+type Customer struct {
+	ID                       string       `pb:"1" json:"id"`
+	BankID                   string       `pb:"2" json:"bank_id"`
+	CustomerNumber           string       `pb:"3" json:"customer_number"`
+	LegalName                string       `pb:"4" json:"legal_name"`
+	PhoneNumber              string       `pb:"5" json:"phone_number"`
+	Email                    string       `pb:"6" json:"email"`
+	FaceImage                FaceImage    `pb:"7" json:"face_image"`
+	DateOfBirth              time.Time    `pb:"8" json:"date_of_birth"`
+	RelationshipStatus       string       `pb:"9" json:"relationship_status"`
+	CreditRating             CreditRating `pb:"10" json:"credit_rating"`
+	CreditLimit              types.Amount `pb:"11" json:"credit_limit"`
+	HighestEducationAttained string       `pb:"12" json:"highest_educational_attained"`
+	EmploymentStatus         string       `pb:"13" json:"employment_status"`
+	KYCStatus                bool         `pb:"14" json:"kyc_status"`
+	LastOKDate               time.Time    `pb:"15" json:"last_ok_data"`
+	Title                    string       `pb:"16" json:"title"`
+	BranchID                 string       `pb:"17" json:"branch_id"`
+}
+
+type CreateCustomerRequest struct {
+	BankID                   string       `pb:"1" json:"bank_id"`
+	LegalName                string       `pb:"2" json:"legal_name"`
+	MobilePhoneNumber        string       `pb:"3" json:"mobile_phone_number"`
+	Email                    string       `pb:"4" json:"email"`
+	FaceImage                FaceImage    `pb:"5" json:"face_image"`
+	DateOfBirth              time.Time    `pb:"6" json:"date_of_birth"`
+	RelationshipStatus       string       `pb:"7" json:"relationship_status"`
+	DobOfDependants          []time.Time  `pb:"8" json:"dob_of_dependants"`
```

Comment:
change to `dependents` the same comment @jwangsadinata mentioned for me to account-applications !! as these are same fields.